lib/widgets/dialog.dart

```diff
@@ -3,28 +3,20 @@ import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/zulip_localizations.dart';
 import 'package:flutter/foundation.dart';
 
-Widget _materialDialogActionText(String text) {
-  return Text(
-    text,
-
-    // As suggested by
-    //   https://api.flutter.dev/flutter/material/AlertDialog/actions.html :
-    // > It is recommended to set the Text.textAlign to TextAlign.end
-    // > for the Text within the TextButton, so that buttons whose
-    // > labels wrap to an extra line align with the overall
-    // > OverflowBar's alignment within the dialog.
-    textAlign: TextAlign.end,
-  );
-}
-
 /// A platform-appropriate action for [AlertDialog.adaptive]'s [actions] param.
 Widget _adaptiveAction({required VoidCallback onPressed, required String text}) {
   switch (defaultTargetPlatform) {
     case TargetPlatform.android:
     case TargetPlatform.fuchsia:
     case TargetPlatform.linux:
     case TargetPlatform.windows:
-      return TextButton(onPressed: onPressed, child: _materialDialogActionText(text));
+      // As suggested by
+      //   https://api.flutter.dev/flutter/material/AlertDialog/actions.html :
+      // > It is recommended to set the Text.textAlign to TextAlign.end
+      // > for the Text within the TextButton, so that buttons whose
+      // > labels wrap to an extra line align with the overall
+      // > OverflowBar's alignment within the dialog.
+      return TextButton(onPressed: onPressed, child: Text(text, textAlign: TextAlign.end));
```

Comment:
nit: There's a lot happening on this line of code. Let's arrange it so the comment is on just the specific thing that it's about, like it was before:

```dart
    // As suggested by
    //   https://api.flutter.dev/flutter/material/AlertDialog/actions.html :
    // > It is recommended to set the Text.textAlign to TextAlign.end
    // > for the Text within the TextButton, so that buttons whose
    // > labels wrap to an extra line align with the overall
    // > OverflowBar's alignment within the dialog.
    textAlign: TextAlign.end,
```

Reply:
alrighty I think I've fixed up all those nits